tests/foreman/cli/test_errata.py

```diff
@@ -459,6 +459,36 @@ def test_negative_install_by_hc_name_without_org_info(self):
             )
         self.assertIn('Error: Could not find organization', context.exception.stderr)
 
+    @tier3
+    @upgrade
+    def test_positive_list_affected_chosts(self):
+        """View a list of affected content hosts for an erratum
+
+        :id: 3b592253-52c0-4165-9a48-ba55287e9ee9
+
+        :Setup: Errata synced on satellite server.
+
+        :Steps: host list --search "applicable_errata = <erratum_id>"
+            --organization-id=<org_id>
+
+        :expectedresults: List of affected content hosts for an erratum is
+            displayed.
+
+        :CaseAutomation: automated
+
+        """
+        result = Host.list(
+            {
+                'search': 'applicable_errata = {}'.format(FAKE_2_ERRATA_ID),
```

Comment:
Up to you, but you may use f-string now:
```python
'search': f'applicable_errata = {FAKE_2_ERRATA_ID}',
```

Reply:
oh yes. will do.